examples/osgi-test-example-mvn/org.osgi.test.example.player.impl/.project

```diff
@@ -11,13 +11,13 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<name>bndtools.core.bndbuilder</name>
```

Comment:
Why are you breaking the osgi-test-example-mvn project Here?

Reply:
used a wrong git add filter